api/data_explorer/__main__.py

```diff
@@ -14,15 +14,18 @@
 from elasticsearch.client.cat import CatClient
 from elasticsearch.exceptions import ConnectionError
 from elasticsearch.exceptions import TransportError
-from elasticsearch_dsl import HistogramFacet
 from elasticsearch_dsl import FacetedSearch
+from elasticsearch_dsl import HistogramFacet
```

Comment:
Let's add a "Samples Overview facet" e2e test.

1. Assert facet before click
2. Assert facet after click
3. Send to Saturn, confirm redirected to import-data. See "Export to Saturn - selected cohort" test.

3 actually fails for me when I manually test it.

Reply:
Added a test, still need to finish the export part.